handbook/growth.md

```diff
@@ -85,6 +85,44 @@ Once a relevant sponsorship opportunity and its prospectus are reviewed:
  
 7. Once the above steps are completed, use the [Speaking events issue template](https://github.com/fleetdm/confidential/issues/new?assignees=mike-j-thomas&labels=&template=6-speaking-event.md&title=Speaking+event) to prepare speakers and participants for the event.
 
+## Salesforce contributor experience check ups
```

Comment:
Please reference this in the "Rituals" table with its DRI and frequency.

Reply:
Can you link to this heading from the table?

Reply:
@mikermcneil - done (I think I did it correctly, can't tell from the preview if it's exactly what you want).